doc/project/checklist.md

```diff
@@ -218,6 +218,11 @@ The security council decides whether an additional RTL review of the relevant co
 
 Any approved new features since D2 have been documented and reviewed with DV/SW/FPGA
 
+### DOC_PARAM_IP
+
+In case the IP supports multiple parameterizations (e.g. instantiation parameters or memory maps), the configurations that are going to be verified and signed off as part of the V* stages must be documented in a section called "Supported Parameter Sets".
+If the IP is instantiated with a non-verified parameterization at a later point, D3 and V3 needs to be re-assessed for that configuration.
```

Comment:
s/a non-verified/an unsupported

To keep the terminology consistent.

Reply:
ack - updated!